src/v/resource_mgmt/cpu_scheduling.h

```diff
@@ -65,6 +65,20 @@ class scheduling_groups final {
     }
     ss::scheduling_group archival_upload() { return _archival_upload; }
 
+    std::vector<std::reference_wrapper<const ss::scheduling_group>>
```

Comment:
It's almost tempting to store them as an array.

Reply:
I don't think it mattes too much. This function only gets called once at start-up.